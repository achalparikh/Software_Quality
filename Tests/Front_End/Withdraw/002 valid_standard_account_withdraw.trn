10
01 TONY STARK          00009 00400.00 S 
00