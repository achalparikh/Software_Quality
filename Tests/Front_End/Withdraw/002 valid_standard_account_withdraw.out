Enter user type - "standard" or "admin": 
Logging in as standard user - enter name:
You are now logged in as "TONY STARK"
COMMAND: withdraw - enter amount:
ALERT: 0.10 non student fee will be deducted. 
Withdrawal of 400.00 - complete!
You are now logged out 
